### PR TITLE
adding jest testing

### PR DESCRIPTION
**Pull request by smontana30**
Fixes #

## Types of changes

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other 

## Checklist:

- [ ] My code follows the code style of this project found on [ESLint](../functions/.eslintrc.yml).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by BautistaBertolami — APPROVED** [review]
Looks good, as far as I am aware. (I'm not an expert on any of this so you might wanna have @HanielDiaz  review it before you merge it)

File: functions/src/Test/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+	// 	// reset our database
+	// 	// eslint-disable-next-line @typescript-eslint/no-floating-promises
+	// 	admin
+	// 		.database()
+	// 		.ref('users')
+	// 		.remove();
+	// });
+
+	it('so see if testing works', async () => {
```
Comment (HanielDiaz) [inline functions/src/Test/index.test.ts]: this should be it('should do X') or something. That's the standard.

File: functions/src/Test/index.test.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+// const testEnv = test();
+// const config = {
+// 	apiKey: process.env.apiKey,
+// 	authDomain: process.env.authDomain,
+// 	databaseURL: process.env.databaseURL,
+// 	projectId: process.env.projectId,
+// 	storageBucket: process.env.storageBucket,
+// 	messagingSenderId: process.env.messagingSenderId,
+// 	appId: process.env.appId
+// };
```
Comment (HanielDiaz) [inline functions/src/Test/index.test.ts]: Why is this config stuff here?

File: functions/src/Test/index.test.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+// const test = require('firebase-functions-test')();
+// const key = functions.config();
+
+// test.mockConfig({ stripe: { key: '23wr42ewr34' } });
```
Comment (HanielDiaz) [inline functions/src/Test/index.test.ts]: why is any of this here?

File: functions/src/Test/index.test.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+	// 	adminStub.mockRestore();
+	// 	test().cleanup();
+
+	// 	// reset our database
+	// 	// eslint-disable-next-line @typescript-eslint/no-floating-promises
+	// 	admin
+	// 		.database()
+	// 		.ref('users')
+	// 		.remove();
+	// });
```
Comment (HanielDiaz) [inline functions/src/Test/index.test.ts]: Is any of this necessary?

**Review by HanielDiaz — CHANGES_REQUESTED** [review]

**Pull request closed without merging**